android/build.gradle

```diff
@@ -94,7 +92,6 @@ dependencies {
     testImplementation "org.assertj:assertj-core:$assertj_version"
 
     testImplementation "io.mockk:mockk:$mockk_version"
-    testImplementation "com.revenuecat.purchases:purchases-store-google:$purchases_version"
```

Comment:
I've removed these dependencies since they seemed unnecessary. CC @vegaro, in case you have more context on why these were needed.

Reply:
I think we only needed `PlatformInfo` in the hybrids, which is now exposed as part of `Purchases`. We will need to fix this in the hybrids.